presto-main/src/main/java/com/facebook/presto/sql/ExpressionUtils.java

```diff
@@ -243,6 +250,14 @@ public static Expression stripDeterministicConjuncts(Expression expression)
                 .collect(toImmutableList()));
     }
 
+    public static Expression stripDynamicFilters(Expression expression)
```

Comment:
I do not like idea of extending `ExpressionUtils` junk drawer. Other option would be to put those `DynamicFilter` related methods to `DynamicFilter` class itselft. Or create `DynamicFilter` specific util class aside of it (`DynamicFilterUtils`/`DynamicFilters`). What do you think?